### PR TITLE
Add exercise style @kyunwang

### PR DESCRIPTION
**Pull request by kyunwang**
The old code will also be pushed as the previous pull request has yet to be accepted

### CODE REVIEW

**Comment by wooorm** [comment]
I’ve merged your other PR, so now there are conflicts?!

File: site/class-2-style/kyunwang/.eslintrc.json [hunk 1]
```diff
@@ -0,0 +1,29 @@
+{
```
Comment (wooorm) [inline site/class-2-style/kyunwang/.eslintrc.json]: Could you remove this file? I’d like to keep our repo clean!

File: site/class-2-debug/kyunwang/index.css [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+  font-family: Roboto, -apple-system, BlinkMacSystemFont, Helvetica Neue, Segoe UI, Oxygen, Ubuntu, Cantarell, Open Sans, sans-serif;
+  max-width: 100%;
+}
+
+body {
+  margin: 0;
+}
+
+svg {
+  width: 100w;
```
Comment (wooorm) [inline site/class-2-debug/kyunwang/index.css]: w > vw

File: site/class-2-debug/kyunwang/index.css [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+  height: 100h;
```
Comment (wooorm) [inline site/class-2-debug/kyunwang/index.css]: h > vh

File: site/class-2-debug/kyunwang/index.css [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+}
+
+.axis path {
+  fill: no;
```
Comment (wooorm) [inline site/class-2-debug/kyunwang/index.css]: no > none

File: site/class-2-debug/kyunwang/index.html [hunk 5]
```diff
@@ -0,0 +1,11 @@
+<!doctype html>
+<!--Based on https://bl.ocks.org/mbostock/3887118 by Mike Bostock-->
+<meta charset=utf8>
+<!--Prevent favicon error from showing up-->
+<link rel="shortcut icon" type=image/x-icon href=data:image/x-icon;,>
+<title>Debug starter code</title>
+<meta content=width=device-width,initial-scale=1 name=viewport>
+<link rel=stylesheet href=../index.css>
```
Comment (wooorm) [inline site/class-2-debug/kyunwang/index.html]: ../index.css > index.css

**Comment by kyunwang** [comment]
Yes for some reason there were merge conflict with untouched files.
The eslint config has been deleted however There are starter files for the debug exercise included in this pull request

File: site/class-2-debug/kyunwang/index.html [hunk 6]
```diff
@@ -0,0 +1,11 @@
...
+<svg></svg>
+<script src=https://d3js.org/d3.v3.min.js></script>
+<script src=../index.js></script>
```
Comment (wooorm) [inline site/class-2-debug/kyunwang/index.html]: ../index.js > index.js

File: site/class-2-debug/kyunwang/index.js [hunk 7]
```diff
@@ -0,0 +1,127 @@
+'use strict';
+
+/* Based on https://bl.ocks.org/mbostock/3887118 by Mike Bostock. */
+
+/* Define margins and size — https://bl.ocks.org/mbostock/3019563. */
+var margin = {top: 48, right: 48, bottom: 48, left: 48};
+var width = 960 - margin.l - margin.r;
+var height = 500 - margin.t - margin.b;
```
Comment (wooorm) [inline site/class-2-debug/kyunwang/index.js]: margin.l -> margin.left ...etc

File: site/class-2-debug/kyunwang/index.js [hunk 8]
```diff
@@ -0,0 +1,127 @@
...
+var yAxis = d3.svg.axis().scale(y).orient('left');
+
+/* Size SVG. */
+var svg = d3
+  .select('svg')
+  .attr('viewBox', [
+    0,
+    0,
+    margin.l + width + margin.r,
+    margin.t + height + margin.b
```
Comment (wooorm) [inline site/class-2-debug/kyunwang/index.js]: Fix the margins here too.

File: site/class-2-debug/kyunwang/index.js [hunk 9]
```diff
@@ -0,0 +1,127 @@
...
+  ].join(' '))
+  .append('g')
+  .attr('transform', 'translate(' + margin.l + ',' + margin.t + ')');
+
+/* Load data. */
+d3.tsv('index.tsv', row, onload);
```
Comment (wooorm) [inline site/class-2-debug/kyunwang/index.js]: tsk > csv.

**Comment by wooorm** [comment]
Dude! You’re doing all the work on one branch, but you should use separate branches and PRs!

**Comment by kyunwang** [comment]
I just started working on the debug exercise xd

**Comment by kyunwang** [comment]
should be okay now. The debug starter files have been removed and eslint config too

**Pull request closed without merging**